admin_app/src/app/dashboard/types.ts

```diff
@@ -0,0 +1,4 @@
+type Period = "day" | "week" | "month" | "year";
+type TimeFrame = "Last 24 hours" | "Last week" | "Last month" | "Last year";
+
+export type { Period, TimeFrame };
```

Comment:
Maybe we should call this `utils.ts` to keep consistency accross modules going forward. Then we will be able to add `tabLabels` from `TabPanel.tsx`.

Reply:
I want to keep types together and utils together. I'll have a think about this